dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -37,12 +37,21 @@
 _dali_tf = _dali_tf_module.dali
 
 def DALIIteratorWrapper(pipeline = None, serialized_pipeline = None, sparse = [],
-                        shapes = [], dtypes = [], batch_size = -1, **kwargs):
+                        shapes = [], dtypes = [], batch_size = -1, prefetch_queue_depth=2, **kwargs):
   """
 TF Plugin Wrapper
 
 This operator works in the same way as DALI TensorFlow plugin, with the exception that is also accepts Pipeline objects as the input and serializes it internally. For more information, please look **TensorFlow Plugin API reference** in the documentation.
   """
+  if type(prefetch_queue_depth) is dict:
+      exec_separated = True
+      cpu_prefetch_queue_depth = prefetch_queue_depth["cpu_size"]
+      gpu_prefetch_queue_depth = prefetch_queue_depth["gpu_size"]
+  elif type(prefetch_queue_depth) is int:
+      exec_separated = False
+      cpu_prefetch_queue_depth = 2 # dummy: wont' be used
```

Comment:
```suggestion
      cpu_prefetch_queue_depth = -1 # dummy: wont' be used
```
If it's dummy, maybe `-1` would better reflect it? And possibly fail, if it turns out, that it not that dummy IRL

Reply:
Done